src/core_plugins/console/public/src/input.js

```diff
@@ -178,8 +178,13 @@ export function initializeInput($el, $actionsEl, $copyAsCurlEl, output) {
 
             var warnings = xhr.getResponseHeader("warning");
             if (warnings) {
-              warnings = _.map(warnings.split(", "), function (warning) {
-                  return "#! Deprecation: " + warning;
+              // pattern for valid warning header
+              var re = /\d{3} [^ ]+ \"([^"]*)\"( \"[^"]*\")/
```

Comment:
Ah you've just pointed out a flaw in my regex.  Not such a master now! In fact it should be:

    "((?:[^"\\]|\\.)*)"

Which means: 
* match `"`
* capture:
** zero or more times: match any character except `\` or `"` or match `\` followed by any character
* match `"`

So it'd match this correctly:

    "Here is an embedded \\ and a \" quote and \n"

That handles any backslashed character, not just quotes